app/views/revisions/_breadcrumbs.html.erb

```diff
@@ -0,0 +1,21 @@
+<nav>
+  <ol class="breadcrumb">
+    <% if @class_name == 'card' %>
+      <li class="breadcrumb-item"><%= link_to 'Methodologies', project_boards_path(current_project) %></li>
+      <li class="breadcrumb-item"><%= link_to "#{@record.board.name} - #{@record.list.name}", project_board_path(current_project, @record.board) %></li>
+      <li class="breadcrumb-item"><%= link_to @record.name, project_board_list_card_path(current_project, @board, @list, @record) %></li>
+    <% elsif @class_name == 'evidence' %>
+      <li class="breadcrumb-item"><a href="javascript:void(0)" data-behavior="sidebar-toggle">Nodes</a></li>
+      <li class="breadcrumb-item"><%= link_to @record.node.label, project_node_path(current_project, @record.node, tab: "#{@class_name.pluralize(2)}-tab") %></li>
```

Comment:
no value in interpolating this variable, you're already in the Evidence section, so hard-code: `tab: 'evidence-tab'`